apps/auth_zero/models.py

```diff
@@ -26,6 +27,15 @@ class User(AbstractUser):
     """Custom Auth User model."""
 
     default_user_type = UserTypes.PATIENT
+    phone_regex = RegexValidator(
+        regex=r"^\+?1?\d{9,15}$",
+        message="""Phone number must be entered in the format:
+         '+999999999'. Up to 15 digits allowed.""",
+    )
+    mobile = models.CharField(validators=[phone_regex], max_length=17)
+    mobile_is_verified = models.BooleanField(default=False)
```

Comment:
do we really need this if we have `otp_verified`?

Reply:
I kind of added that because it was specified in the list of custom fields(on jira) to be added. If it's not useful, I can as well take that out.

Reply:
yes remove it.